stdlib/testing/testdata/time_hour.flux

```diff
@@ -0,0 +1,71 @@
+package testdata_test
+
+import "testing"
+import "calendar"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,string,string,double
+#group,false,false,false,true,true,false
+#default,_result,,,,,
+,result,table,_time,_measurement,_field,_value
+,,0,2018-05-22T19:53:00Z,_m,FF,1
+,,0,2018-05-22T19:53:10Z,_m,FF,1
+,,0,2018-05-22T19:53:20Z,_m,FF,1
+,,0,2018-05-22T19:53:30Z,_m,FF,1
+,,0,2018-05-22T19:53:40Z,_m,FF,1
+,,0,2018-05-22T19:53:50Z,_m,FF,1
+,,1,2018-05-22T19:53:00Z,_m,QQ,1
+,,1,2018-05-22T19:53:10Z,_m,QQ,1
+,,1,2018-05-22T19:53:20Z,_m,QQ,1
+,,1,2018-05-22T19:53:30Z,_m,QQ,1
+,,1,2018-05-22T19:53:40Z,_m,QQ,1
+,,1,2018-05-22T19:53:50Z,_m,QQ,1
+,,1,2018-05-22T19:54:00Z,_m,QQ,1
+,,1,2018-05-22T19:54:10Z,_m,QQ,1
+,,1,2018-05-22T19:54:20Z,_m,QQ,1
+,,2,2018-05-22T19:53:00Z,_m,RR,1
+,,2,2018-05-22T19:53:10Z,_m,RR,1
+,,2,2018-05-22T19:53:20Z,_m,RR,1
+,,2,2018-05-22T19:53:30Z,_m,RR,1
+,,3,2018-05-22T19:53:40Z,_m,SR,1
+,,3,2018-05-22T19:53:50Z,_m,SR,1
+,,3,2018-05-22T19:54:00Z,_m,SR,1
+"
+
+outData = "
+#group,false,false,true,true,false,false
+#datatype,string,long,string,string,dateTime:RFC3339,long
+#default,_result,,,,,
+,result,table,_field,_measurement,_time,_value
+,,0,FF,_m,2018-05-22T19:53:00Z,19
+,,0,FF,_m,2018-05-22T19:53:10Z,19
+,,0,FF,_m,2018-05-22T19:53:20Z,19
+,,0,FF,_m,2018-05-22T19:53:30Z,19
+,,0,FF,_m,2018-05-22T19:53:40Z,19
+,,0,FF,_m,2018-05-22T19:53:50Z,19
+,,1,QQ,_m,2018-05-22T19:53:00Z,19
+,,1,QQ,_m,2018-05-22T19:53:10Z,19
+,,1,QQ,_m,2018-05-22T19:53:20Z,19
+,,1,QQ,_m,2018-05-22T19:53:30Z,19
+,,1,QQ,_m,2018-05-22T19:53:40Z,19
+,,1,QQ,_m,2018-05-22T19:53:50Z,19
+,,1,QQ,_m,2018-05-22T19:54:00Z,19
+,,1,QQ,_m,2018-05-22T19:54:10Z,19
+,,1,QQ,_m,2018-05-22T19:54:20Z,19
+,,2,RR,_m,2018-05-22T19:53:00Z,19
+,,2,RR,_m,2018-05-22T19:53:10Z,19
+,,2,RR,_m,2018-05-22T19:53:20Z,19
+,,2,RR,_m,2018-05-22T19:53:30Z,19
+,,3,SR,_m,2018-05-22T19:53:40Z,19
+,,3,SR,_m,2018-05-22T19:53:50Z,19
+,,3,SR,_m,2018-05-22T19:54:00Z,19
```

Comment:
I'd like to see these tests cases have varying output, so instead of using a time stamp with the hour 19 for all values, instead use varying hours. What do you think, should we add a 24 rows, one for each hour?

Reply:
We could also do the same thing for all the other functions as well. The largest would be day of the year which shouldn't be too large.